example/react-app/src/App.tsx

```diff
@@ -156,8 +155,8 @@ function App() {
       uiConsole("web3auth not initialized yet");
       return;
     }
-    const idToken = await web3authSFAuth?.authenticateUser();
-    uiConsole(idToken);
+    const id_token = await web3authSFAuth?.authenticateUser();
```

Comment:
ideally, we should avoid using snake_casing in code.
In JS, its preferred to use camelCase